service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageControllerCitizen.kt

```diff
@@ -0,0 +1,34 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+package fi.espoo.evaka.messaging.message
+
+import fi.espoo.evaka.shared.auth.AuthenticatedUser
+import fi.espoo.evaka.shared.auth.UserRole
+import fi.espoo.evaka.shared.db.Database
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/citizen/messages")
+class MessageControllerCitizen {
+
+    @GetMapping("/my-accounts")
+    fun getAccountsByUser(db: Database.Connection, user: AuthenticatedUser): List<MessageAccount> {
+        user.requireOneOfRoles(UserRole.END_USER)
+        return db.read { it.getMessageAccountsForUser(user) }
+    }
+
+    @GetMapping("/unread")
```

Comment:
Nit: Something like `/unread-count` would probably be slightly more idiomatic.

Reply:
Changed as discussed.